doc/source/rllib/rllib-algorithms.rst

```diff
@@ -671,12 +673,14 @@ Tuned examples: `CartPole-v0 <https://github.com/ray-project/ray/blob/master/rll
    :end-before: __sphinx_doc_end__
 
 
+.. _bandits:
+
 Contextual Bandits
 ~~~~~~~~~~~~~~~~~~
 
 The Multi-armed bandit (MAB) problem provides a simplified RL setting that
-involves learning to act under one situation only, i.e. the state is fixed.
-Contextual bandit is extension of the MAB problem, where at each
+involves learning to act under one situation only, i.e. the observation/state is fixed.
```

Comment:
from this paper (http://rob.schapire.net/papers/www10.pdf) MAB is a special case of contextual bandit where the context (user) and the arms are both fixed.
maybe we can say "i.e., the context and arms are both fixed."

Reply:
Perfect, thanks for the hint and the review @gjoliver ! Will add this before merging.